src/applications/vaos/tests/new-appointment/components/ReviewPage/index.direct.unit.spec.jsx

```diff
@@ -483,7 +484,14 @@ describe('VAOS <ReviewPage> direct scheduling with v2 api', () => {
       status: 'booked',
       locationId: '983',
       clinic: '455',
-      comment: 'Follow-up/Routine: I need an appt',
+      reasonCode: {
+        coding: [
+          {
+            code: 'Follow-up/Routine',
+          },
+        ],
+        text: 'I need an appt',
```

Comment:
delete `text:` It is not required for radio options besides "My reason isn't listed"

Reply:
Done